ports/rp2/mpconfigport.h

```diff
@@ -109,6 +109,7 @@
 #endif
 #elif PICO_RISCV
 #define MICROPY_EMIT_RV32                       (1)
+#define MICROPY_EMIT_RV32_ZBA                   (1)
```

Comment:
This might also need to be enabled in `mpy-cross`, otherwise it won't use the extensions.

But of course that means we probably need an option to `mpy-cross` to enable/disable it, so it can work with targets that don't have this extension.  And this starts to make things quite complicated!

Is it possible to just enable this unconditionally in `mpy-cross`, and yet the CI (qemu tests) still pass??

Reply:
> I thought `mpy-cross` didn't compile `@micropython.viper` functions

Yet, it definitely can.  It can do all of `@micropython.native`, `@micropython.viper` and `@micropython.asm_XXX`.

There are already options like `-march=armv7emsp` that you can pass to `mpy-cross` to select the correct arch, so I guess they need to be extended to select sub-features of RV32.

But that's definitely out of scope for this PR.

Reply:
I'm aware of `mpy-cross` compiling inline asm code, I just assumed that viper code was geared towards interactive/compiled-at-runtime usage and thus left alone.  Good to know though :)

I've removed the RV32 code from this PR so it can be part of another one that will also bring in more extensive `mpy-cross` changes.